test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -9,6 +9,8 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
```

Comment:
Sorry, but I decided it's irrational to spend time to please the GHC here.

Reply:
Where even was the issue?

Reply:
Ok, adding a type hint worked properly this time.